client/i18n/locales/english/translations.json

```
@@ -101,7 +101,66 @@
         "testimony": "\"I've always struggled with learning JavaScript. I've taken many courses but freeCodeCamp's course was the one which stuck. Studying JavaScript as well as data structures and algorithms on <strong>freeCodeCamp gave me the skills</strong> and confidence I needed to land my dream job as a software engineer at Spotify.\""
       }
     },
-    "certification-heading": "Earn free verified certifications in:"
+    "certification-heading": "Earn free verified certifications in:",
+    "faq": "Frequently asked questions:",
+    "faqs": [
+      {
+        "question": "What exactly is freeCodeCamp?",
+        "answer": [
+          "freeCodeCamp is a community of people from all around the world who are learning to code together. We're a 501(c)(3) public charity."
+        ]
+      },
+      {
+        "question": "How will freeCodeCamp help me learn to code?",
+        "answer": [
+          "You will learn to code by building dozens of projects, step-by-step, right in your browser, code editor, or mobile app.",
+          "You will also earn free verified certifications along the way."
```

This looks out of the norm with it being two sentences.

```suggestion
          "You will learn to code by building dozens of projects, step-by-step, right in your browser, code editor, or mobile app. You will also earn free verified certifications along the way."
```

---

This is how Quincy wrote it up - so I did exactly as he had it.